app/src/main/java/com/simplecity/amp_library/utils/MusicUtils.java

```diff
@@ -30,35 +30,35 @@
         int NEW_PLAYLIST = 2;
     }
 
+    /**
+     * Passes along a list of songs, wrapped in a Single obj, to be played, starting at position 0.
```

Comment:
Better to comment with javadoc link, rather than referring to `Single` as 'Single obj'.
```
/**
*  Passes along a list of songs, wrapped in an {@link Single}, to be played, starting at position 0.
**/
```